style.css

```diff
@@ -0,0 +1,30 @@
+/*
+   This is a CSS comment!
+   They are ignored by the browser.
+   I'll leave you notes in here using these!
+
+   TODO: Annotate a CSS demo...
+*/
```

Comment:
Just a note to @lemonhawk21 that @amiller-gh knows like EVERY CSS property...but that's not normal. He's a CSS (and general coding) wizard so keep that in mind when you need to look things up that he somehow just knows.

Reply:
![download](https://user-images.githubusercontent.com/7856443/35532138-083a0d7e-04ee-11e8-81db-76ed49ab88f5.jpg)